### PR TITLE
Releases/v0.3.0

### PR DESCRIPTION
**Pull request by daytime-em**
# API Changes

* `MuxUpload`'s initializer no longer requires a MIME type or Retry Time. These are calculated internally
* Added methods for querying the `UploadManager` for the list of currenty-active uploads, and listening for changes to the list

## Improvements

* NFC: call into upload manager reference (#14)
* api: Remove extraneous MIME type and retry time config fields, add opt-out for event tracking (#16)
* doc: Add a much-improved example app (#15)

## Fixes
* Fix: Uploads continue without reporting progress after being resumed (#24)
* Fix: handlers not retained unless callers retain the `MuxUpload` (#25)

Co-authored-by: AJ Lauer Barinov <102617203+andrewjl-mux@users.noreply.github.com>
Co-authored-by: Liam Lindner <liam@mux.com>
Co-authored-by: Emily Dixon <edixon@mux.com>
Co-authored-by: GitHub <noreply@github.com>

### CODE REVIEW

File: Sources/MuxUploadSDK/InternalUtilities/UploadEvent.swift [hunk 1]
```diff
@@ -43,7 +43,7 @@ struct UploadEvent: Codable {
 
         self.deviceModel = device.model
 
-        self.appName = Bundle.main.appName
```
Comment (andrewjl-mux) [inline Sources/MuxUploadSDK/InternalUtilities/UploadEvent.swift:46]: Does this result in a different value for `appName` for the same app?

File: Sources/MuxUploadSDK/PublicAPI/Version.swift [hunk 2]
```diff
@@ -11,9 +11,9 @@ public struct Version {
   /// Major version.
   public static let major = 0
   /// Minor version.
-  public static let minor = 2
+  public static let minor = 3
```
Comment (andrewjl-mux) [inline Sources/MuxUploadSDK/PublicAPI/Version.swift:14]: Thanks for remembering to change these, we should look into adding a script for automating this change

File: Sources/MuxUploadSDK/InternalUtilities/UploadEvent.swift (hunk 1 above)
Comment (daytime-em) [reply Sources/MuxUploadSDK/InternalUtilities/UploadEvent.swift:46]: It does, but this is known. We want the unique name because it's easier to identify

**Review by andrewjl-mux — APPROVED** [review]
🚢

File: Sources/MuxUploadSDK/PublicAPI/Version.swift (hunk 2 above)
Comment (daytime-em) [reply Sources/MuxUploadSDK/PublicAPI/Version.swift:14]: I wonder if swift-syntax could help us

File: Sources/MuxUploadSDK/InternalUtilities/UploadEvent.swift (hunk 1 above)
Comment (andrewjl-mux) [reply Sources/MuxUploadSDK/InternalUtilities/UploadEvent.swift:46]: ok, just something to note if we're ever trying to correlate upload stats across versions

**Pull request merged**